src/core/client/api.ts

```diff
@@ -298,13 +309,42 @@ export interface CreateBuilder {
   flow(name: string): Promise<Site>;
   service(name: string): Promise<Site>;
   decision(name: string): Promise<Site>;
+  branch(body: AstCommand[]): Promise<Site>;
 }
 
 export interface DeleteBuilder {
   tag(tagId: TagId): Promise<Site>;
   flow(flowId: FlowId): Promise<Site>;
   service(serviceId: ServiceId): Promise<Site>;
   decision(decisionId: DecisionId): Promise<Site>;
+  branch(branchId: BranchId): Promise<Site>;
+}
+
+export interface TagDiff {
```

Comment:
Naming is a bit inconsistent.
DIFFRequest vs TagDIFF.
SummaryItem vs AstTagSummary.